target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -59,6 +62,10 @@ mediatek_setup_macs()
 		wan_mac=$(mtd_get_mac_ascii Bdata ethaddr_wan)
 		label_mac=$wan_mac
 		;;
+	zyxel,ex5601-t0)
+		lan_mac=$(mtd_get_mac_binary "Factory" 0x2A)
+		wan_mac=$(mtd_get_mac_binary "Factory" 0x24)
+		;;
```

Comment:
Is nvmem cell not supported by the network driver? considering Factory is a partition, data is binary... no sense to use board.d script when you can use dts directly to reference them.

Same comment for the wifi

Reply:
Can you link and example for DTS directly? I can try it with initramfs and switch to DTS... I ve used the same method of the Zyxel firmware, they map lan and wan on the board.d scripts.

Reply:
Here is an example:
https://github.com/openwrt/openwrt/blob/master/target/linux/mediatek/dts/mt7981b-cudy-wr3000-v1.dts#L225

P.S. And `0x2a` without the caps would be more traditional for this project. :)

Reply:
While that may work for gmac0 and gmac1, what about the wifi? Without the wifi Mac fix the AP 2.4 takes the right Mac address but the wifi 5 is taking a random one, hence the need to fix it in wifi2.4 Mac+1

Reply:
Team, I did like you suggested, I removed the board.d configs and added mac address configurations into gmacs. Everything works as expected but the Wifi. Like I mentioned in my previous comment, the two wifi have 2 different mac addresses where wifi 2.4 ghz is 0x0004 from factory and wifi 5.ghz is 0x0004 +1. With the current commit wifi 2.4 and 5 have the same mac-address which is not correct. What should I use to fix the wifi 5ghz mac address?

Reply:
Of course you can use `base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac` for fixing 5ghz MAC.

Reply:
Done